assets/js/base/components/quantity-selector/style.scss

```diff
@@ -12,12 +12,13 @@
 
 .wc-block-components-quantity-selector {
```

Comment:
To fix this I've removed the margin from the `last-of-type` of `.wc-block-components-product-details` and added margin to the bottom of `.wc-block-components-product-metadata` - do you think this was the right approach? I had to add `0.75em` of margin because the Remove item text is quite small, so the gap looked bigger than the gap above.